source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -0,0 +1,367 @@
+method to modify the way the driver manages its' connection pool.
+
+The following table describes all the methods you can chain to your
+settings to modify the driver's behavior:
+
+.. list-table::
+   :widths: 40 60
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``addConnectionPoolListener​()``
+     - Adds a listener for connection pool-related events.
+
+   * - ``applyConnectionString​()``
+     - Uses the settings from a ``ConnectionString`` object.
+
+   * - ``applySettings()``
+     - Uses the connection pool settings specified in a ``ConnectionPoolSettings`` object.
+
+   * - ``maintenanceFrequency()``
+     - Sets the frequency for running a maintenance job.
+
+   * - ``maintenanceInitialDelay()``
+     - Sets the time to wait before running the first maintenance job.
+
+   * - ``maxConnectionIdleTime()``
+     - Sets the amount of time a connection can be idle before it's closed.
+
+   * - ``maxConnectionLifeTime()``
+     - Sets the maximum time a pooled connection can live for.
+
+   * - ``maxSize()``
+     - Sets the maximum amount of connections associated with a connection pool.
+
+   * - ``maxWaitTime()``
+     - Sets the maximum time to wait for an available connection.
+
+   * - ``minSize()``
+     - Sets the minimum amount of connections associated with a connection pool.
+
+Example
+~~~~~~~
+
+This example specifies the following driver behavior in a pool of
+``Connection`` types:
+
+- The thread to wait at most ``3 MINUTES`` for an available connection
+- To have at most ``200`` connections associated with the pool
+
+.. literalinclude:: /includes/fundamentals/code-snippets/mcs.java
+   :start-after: begin ConnectionPoolSettings
+   :end-before: end ConnectionPoolSettings
+   :language: java
+   :emphasize-lines: 3-5
+   :dedent:
+
+.. _mcs-server-settings:
+
+Server Settings
+---------------
+
+Chain the `applyToServerSettings() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.Builder.html#applyToServerSettings(com.mongodb.Block)>`__
+method to modify the driver's behavior when monitoring each MongoDB
+server.
+
+The following table describes all the methods you can chain to your
+settings to modify the driver's behavior:
+
+.. list-table::
+   :widths: 40 60
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``addServerListener()``
+     - Adds a listener for server-related events.
+
+   * - ``addServerMonitorListener​()``
+     - Adds a listener for server monitor-related events.
+
+   * - ``applyConnectionString​()``
+     - Uses the settings from a ``ConnectionString`` object.
+
+   * - ``applySettings()``
+     - Uses the server settings specified in a ``ServerSettings`` object.
+
+   * - ``heartbeatFrequency()``
+     - Sets the interval for a cluster monitor to attempt reaching a server.
+
+   * - ``minHeartbeatFrequency()``
+     - Sets the interval for server monitoring checks.
```

Comment:
The description in the javadoc is more accurate, though not very informative either

Reply:
Perhaps "minimum interval"?